05.ls/ls01.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+ROW_NUM = 3
+ROW_MAX_WIDTH = 24
+ALL_FILES = Dir.glob('*').sort
+
+def main
+  column_num = ALL_FILES.length / ROW_NUM
+  files = get_transposed_all_files(column_num)
+  display(files)
+end
+
+def get_transposed_all_files(column_num)
+  ALL_FILES.push(' ') while ALL_FILES.length % ROW_NUM != 0
+  transposed_files = ALL_FILES.each_slice(column_num).to_a.transpose
+  transposed_files.first(column_num).each do |each_column|
+    ROW_NUM.times do |index|
+      each_column[index] += ' ' * (ROW_MAX_WIDTH - each_column[index].length)
+    end
+  end
+end
+
+def display(files)
+  files.each do |each_files|
```

Comment:
`each`とかでループする時のブロックパラメータは、コレクション（配列）の変数名を単数形にしたものにするのが定石です。
```suggestion
  files.each do |file|
```

Reply:
よく考えたら`file`の中には3つのファイル名が入ってるんですね。
3つ入ってるのに変数名が`file`なのはおかしい！ともし思うようであれば、そもそも`files`の方の変数名を考え直した方が良いかもしれません。